test/units/testsuite-43.sh

```diff
@@ -6,6 +6,11 @@ set -o pipefail
 # shellcheck source=test/units/util.sh
 . "$(dirname "$0")"/util.sh
 
+if [[ "$(sysctl -ne kernel.apparmor_restrict_unprivileged_userns)" == "1" ]]; then
```

Comment:
```suggestion
if [[ "$(sysctl -ne kernel.apparmor_restrict_unprivileged_userns)" -eq 1 ]]; then
```

("" evaluates to 0 in arithmetic expressions, so it's safe to use here)